docs/manual-EN/3.build-develop-and-administration/5.storage-service-administration/data-export/dump-tool.md

```diff
@@ -6,7 +6,7 @@ Dump Tool is a single-machine off-line data dumping tool that can be used to dum
 
 The source code of the dump tool is under `nebula/src/tools/db_dump` directory. You can use command `make db_dump` to compile it. Before using this tool, you can use the [SHOW HOSTS](../../../2.query-language/4.statement-syntax/3.utility-statements/show-statements/show-hosts-syntax.md) statement in the **Nebula Graph** CLI to check the distribution of the partitions. Also, you can use the `vertex_id % partition_num` statement to calculate which partition the vertex's corresponding [key](../../../1.overview/3.design-and-architecture/2.storage-design.md) is located.
 
-**Note:** The _Dump Tool_ is located in the rpm package and its directory is `nebula/bin/`. Since the tool dumps data by opening the RockDB, you need to use it on the machines that have the storage service deployed and make sure the meta_server is started. Please refer to the following section on detailed usage.
+**Note:** The _Dump Tool_ is located in the rpm package and its directory is `nebula/bin/`. Since this tool dumps data by directly opening the RockDB, you need to use it offline. Stop the storaged process and make sure the meta_server is started. Please refer to the following section on detailed usage.
```

Comment:
"Since this tool dumps data by directly opening the RockDB, you need to use it offline. Stop the storaged process and make sure the meta_server is started. "==> cannot get it. besides, use "because", not "since"; "storaged process" ==> "storaged service"? "meta_server" ==> "metad service"?

Reply:
"storaged process" ==> "storaged service" I don't think so. When you start a service, there is a process in your machine. You need to stop it.

"meta_server" ==> "metad service". I agree with this